.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and test CDS
+
+on:
+  push:
+    branches:
+  # pull_request:
+  #   branches:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # The first two permissions are needed to interact with GitHub's OIDC Token endpoint.
+    # The second set of two permissions are needed to write test results back to GH
+    permissions:
+      id-token: write
+      contents: read
+      checks: write
+      pull-requests: write
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # GITHUB_RUN_NUMBER always starts from 1, so to maintain compatibility we adjust it here
+      - name: Make GITHUB_RUN_NUMBER env var available outside of shells
+        working-directory: ${{env.GITHUB_WORKSPACE}}
+        shell: bash
+        run: |
+          echo GITHUB_RUN_NUMBER=$(( GITHUB_RUN_NUMBER + 91 ))>> $GITHUB_ENV
```

Comment:
Might be a good idea to increase this number from 91 as we are already higher than that at the moment in Team City.